src/pwncore/db/__init__.py

```diff
@@ -0,0 +1,126 @@
+from tortoise import fields
+from tortoise import Model
+#from passlib.hash import bcrypt
+
+
+class Participant(Model):
+    name : str = fields.TextField(null=False)
+    registrationNo : fields.CharField = fields.CharField(max_length=40, pk=True)
+    email : fields.CharField = fields.CharField(max_length=255, null=True, unique=True)
+    phoneNo : fields.CharField = fields.CharField(max_length=32, null=True, unique=True)
+
+    async def email_fn(self, get=False, update=False, new_value=None):
```

Comment:
Why do these `*_fn` functions exist?
what are they doing?

Reply:
Just written those for test cases and to create a base model .. will be rewritten aft getting a go on the model ...